app/models/dropdown_option.rb

```diff
@@ -10,6 +10,6 @@
 #
 
 class DropdownOption < ApplicationRecord
-  has_one :child_question, class_name: 'Question', as: :parent_option, :dependent => :destroy
+  has_many :child_questions, class_name: 'Question', as: :parent_option, :dependent => :destroy
```

Comment:
@HacksForHugs Should RangeOption be kept consistent, i.e. has_many child questions? I would also need to change my PR code.

Reply:
waitaminute i do not recall ever making this change